gridstatus/eia.py

```diff
@@ -727,6 +752,28 @@ def _handle_fuel_type_data(df):
     return df
 
 
+def _handle_henry_hub_natural_gas_spot_prices(df):
+    df["Interval Start"] = pd.to_datetime(df["period"], utc=True)
```

Comment:
The other EIA datasets use "period" as the "Interval End" but that is not correct for this dataset because that would put the data one day ahead of how the EIA shows it [here](https://www.eia.gov/dnav/ng/hist/rngwhhdD.htm).

Reply:
maybe add this as a comment in the code too?